src/release.rs

```diff
@@ -66,13 +68,21 @@ impl BuildConfig {
             debug_build,
             _build_path: build_path,
             solana_root_path: solana_root_path.clone(),
+            release_channel,
         })
     }
 
     pub async fn prepare(&self) -> Result<(), Box<dyn Error>> {
         match self.deploy_method {
             DeployMethod::Tar => {
-                return Err(boxed_error!("Tar deploy method not implemented yet."));
+                let file_name = "solana-release";
```

Comment:
nit: Will this always be hardcoded? If so, it might be simpler to just hardcode it in `setup_tar_deploy`